packages/aikit-ui/src/blocks/Chat.tsx

```diff
@@ -0,0 +1,288 @@
+import { useState, useRef, useEffect } from "react";
+
+import {
+  Conversation,
+  ConversationContent,
+  ConversationScrollButton,
+} from "@hebo/aikit-ui/_ai-elements/conversation";
+import {
+  Message as Message,
+  MessageContent,
+} from "@hebo/aikit-ui/_ai-elements/message";
+import {
+  PromptInput,
+  PromptInputButton,
+  PromptInputModelSelect,
+  PromptInputModelSelectContent,
+  PromptInputModelSelectItem,
+  PromptInputModelSelectTrigger,
+  PromptInputModelSelectValue,
+  PromptInputSubmit,
+  PromptInputTextarea,
+  PromptInputToolbar,
+  PromptInputTools,
+} from "@hebo/aikit-ui/_ai-elements/prompt-input";
+import { Button } from "@hebo/aikit-ui/_shadcn/ui/button";
+
+// Types based on models.schema.json
+type ModelsConfig = {
+  models: Array<{
+    alias: string;
+    type: string;
+    endpoint?: {
+      baseUrl: string;
+      fetch?: (
+        input: RequestInfo | URL,
+        init?: RequestInit,
+      ) => Promise<Response>;
+    };
+  }>;
+};
+
+export function Chat({ modelsConfig }: { modelsConfig: ModelsConfig }) {
+  const [currentModelAlias, setCurrentModelAlias] = useState(
+    modelsConfig.models.length > 0 ? modelsConfig.models[0].alias : "",
+  );
+  const [messages, setMessages] = useState<UIMessage[]>([]);
+  const [input, setInput] = useState("");
+  const [isLoading, setIsLoading] = useState(false);
+
+  // Accessibility refs
+  const promptInputTextareaRef = useRef<HTMLTextAreaElement>(null);
+
+  // Get current model config - only return a model if we have a valid alias and it exists
+  const currentModel = currentModelAlias
+    ? modelsConfig.models.find((m) => m.alias === currentModelAlias)
+    : undefined;
+
+  // Create OpenAI client based on current model (only if model exists)
+  const openai = currentModel
+    ? createOpenAI({
+        apiKey: "",
+        baseURL: currentModel.endpoint?.baseUrl || "",
+        fetch: currentModel.endpoint?.fetch,
+      })
+    : undefined;
+
+  useEffect(() => {
+    const available = modelsConfig.models ?? [];
+
+    if (available.length === 0) {
+      setCurrentModelAlias("");
+    } else if (!available.some((m) => m.alias === currentModelAlias)) {
+      setCurrentModelAlias(available[0].alias);
+    }
+  }, [modelsConfig.models, currentModelAlias]);
+
+  const renderMessagePart = (part: UIMessage["parts"][0]) => {
+    if (part.type === "text") return part.text;
+    if (part.type === "dynamic-tool" && "input" in part) {
+      return JSON.stringify(part.input);
+    }
+    return "";
+  };
+
+  useEffect(() => {
+    const handleGlobalKeyDown = (e: KeyboardEvent) => {
+      // Don't focus if user is already typing in an input
+      if (
+        e.target instanceof HTMLInputElement ||
+        e.target instanceof HTMLTextAreaElement ||
+        (e.target instanceof HTMLElement && e.target.isContentEditable)
+      ) {
+        return;
+      }
+
+      // Only trigger on Ctrl+L (Windows/Linux) or Cmd+L (Mac)
```

Comment:
Move shortcuts to the object instead of global handler (as discussed previously)